addons/account/models/account_move.py

```diff
@@ -286,6 +286,15 @@ def _sequence_fixed_regex(self):
         index=True,
         copy=False,
     )
+    delivery_date = fields.Date(
```

Comment:
Shouldn't these fields be defined in the `delivery` module?

Reply:
From the discussion i had with PO, we want the delivery date in the account module because a lot of localisation will need it like HU, SA, PL, etc etc, i'm off until monday but i can check more into this when i come back

Reply:
Then, why not have `l10n_xx_delivery` modules?

Reply:
Cause we will to have new module for all the localisation that will need a delivery date, so we think it's simpler to just put the field in account and all loca that wants it can just overide the display. 
In fact all localisation will have the field in the other info tab, juste the display in the header will be modified since it's for some localisation a legal obligation